src/rgw/rgw_keystone.h

```diff
@@ -6,6 +6,12 @@
 
 #include "rgw_common.h"
 
+int open_cms_envelope(CephContext *cct, string& src, string& dst);
```

Comment:
@rzarzynski please add `rgw_` prefix to all the utility functions that are not static anymore